### ISSUE TITLE
tracemapper matches coordinates to route poorly

### ISSUE THREAD

**Issue by gladysmcgan** [eclipse-sumo/sumo#12533]
I am trying to create a route from a list of coordinates using tracemapper, but tracemapper was not able to get candidate edges for many coordinates. The output route has only several disconnected edges. 

The input coordinates form a dense shape along a valid route, so it was surprising that the matching was not successful. 

The tracemapper configuration and input files are here: [google drive folder](https://drive.google.com/drive/folders/1n0c8CjBFlU5TYFmE5G-x6eT9STD564Jl?usp=sharing) 

I want to use these created routes for pt lines which are not available in OSM, similar topic to #11327 

output route (left), coordinate trace (right)
<img width="320" alt="output_route" src="https://user-images.githubusercontent.com/59922884/215278194-8f6b6c1f-810c-4755-809d-56e8b80bdf3b.png">    ![gis_coords](https://user-images.githubusercontent.com/59922884/215278067-83fe74a3-ad83-42ea-8c03-e542a4ff2ee4.png)

SUMO Version v1_12_0+2004-e7e2d65b133

**Comment by m-kro** [comment]
Could be similar to #12101 ?

**Comment by behrisch** [comment]
No, I think it is rather due to the poor defaults of the tracemapper script. By default it does not try to fix any gaps in the route and has a maximum distance of 1m between the edge and the point to map. Using `-d 10 --fill-gaps 200` already gives much better results

**Comment by namdre** [comment]
should this ticket be promoted to "Feature" or closed?

**Comment by gladysmcgan** [comment]
This reply is very delayed, sorry! I have been using the tip given (-d 10 --fill-gaps 200) since then, and it indeed helps. Thanks :)
